overlord/snapstate/storehelpers.go

```diff
@@ -201,6 +203,13 @@ var installSize = func(st *state.State, snaps []minimalInstallInfo, userID int)
 	return total, nil
 }
 
+func setActionValidationSets(action *store.SnapAction, valsets []string) {
+	for _, vs := range valsets {
+		vskey := strings.Split(vs, "/")
```

Comment:
nitpick: vskey -> validationKeys (or some other name of your liking, but I think the plural is important)

Reply:
Renamed to keyParts (it's splitting a key into /-separated components, so it's still about a single validation key and not keys).